languages/eo.lua

```diff
@@ -1,4 +1,81 @@
 }
+
+SILE.doTexlike([[%
+\define[command=tableofcontents:title:eo]{Enhavoj}%
+\define[command=book:chapter:pre:eo]{Ĉapitro №}%
```

Comment:
This command form of adding string localizations has been completely deprecated in SILE v0.13.0 in favor of Fluent format l10n files. Please move these to `i18n/eo.ftl`; the other files in that folder should make it pretty obvious how to format these strings.

If the № is mandated by or at least normative in Esperanto then of course it's fine; if is optional and just your preference then lets stick with just the plain variant and let users set their preferred localization via `\ftl{}`. Also note if dropping the № this should probably then end in a space.

Reply:
It's required unless I call nth(…), but I wasn't sure how

Reply:
Basically you can write it either "Ĉapitro №*n*" or "Ĉapitro 1a" (2a…*n*a). I honestly don't know which is more common, probably the latter, but it's not like books abound in Esperanto lol. I've seen it before with the ª as well. (Hmm, maybe my fonts should have an <sup>aj</sup> ordinal indicator…)

Reply:
Okay, then leave it for now. I haven't implemented function callbacks in fluent-lua yet. When I do we can update the localization with a callback to the number formatting utility.

The strings still need moving to Fluent though.

Reply:
I was able to fix both issues. I misunderstood the `*:post` macros.